cms/static/js/views/pages/container.js

```diff
@@ -162,6 +170,16 @@ define(["jquery", "underscore", "gettext", "js/views/pages/base_page", "js/views
                 }
             },
 
+            updatePreviewButton: function(show_previews){
```

Comment:
It seems odd that the toggle_previews method is not defined in the same place as updatePreviewButton. Is there any reason for that?

And toggle_previews should be a camelCase to match the rest of the JavaScript methods.